docs/markdown/Reference-manual.md

```diff
@@ -119,7 +119,9 @@ the following:
   environment juggling. *(since 0.52.0)* A dictionary is also accepted.
 - `exe_wrapper`: a list containing the wrapper command or script followed by the arguments to it
 - `gdb`: if `true`, the tests are also run under `gdb`
-- `timeout_multiplier`: a number to multiply the test timeout with
+- `timeout_multiplier`: a number to multiply the test timeout with.
+  *Since 0.57* if timeout_multiplier is `<= 0` the test has infinite duration,
+  previous versions of Meson were failing the test with a timeout immediately.
```

Comment:
```suggestion
  *Since 0.57* if timeout_multiplier is `<= 0` the test has infinite duration,
  in previous versions of Meson the test would fail with a timeout immediately.
```
Writing this in the past perfect feels more natural